README.rst

```diff
@@ -88,7 +89,10 @@ Before running tests, please ensure that following environment variables are set
     ==> BASIC_AUTH_PASSWORD
     ==> USER_LOGIN_EMAIL
     ==> USER_LOGIN_PASSWORD
-
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
 
 To run all the tests:
 
```

Comment:
@estute 
by default we are excluding white label tests on Jenkins:
https://github.com/edx/edx-e2e-tests/blob/master/jenkins/end_to_end_tests.sh#L32

I think we can specify:
- Run all tests 'paver e2e_test'
- With flaky 'paver e2e_test --with-flaky'
- Exclude white label 'paver e2e_test --exclude=whitelabel'
- With flaky without whitelabel 'paver e2e_test --with-flaky --exclude=whitelabel'
Does that make sense?